ui/shared/components/table/RnaSeqJunctionOutliersTable.jsx

```diff
@@ -0,0 +1,115 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { connect } from 'react-redux'
+
+import { getIndividualsByGuid } from 'redux/selectors'
+import { RNASEQ_JUNCTION_PADDING } from 'shared/utils/constants'
+import { camelcaseToTitlecase } from 'shared/utils/stringUtils'
+import { GeneSearchLink } from 'shared/components/buttons/SearchResultsLink'
+import DataTable from 'shared/components/table/DataTable'
+import { COVERAGE_TYPE, JUNCTION_TYPE } from 'shared/components/panel/family/constants'
+import ShowGeneModal from 'shared/components/buttons/ShowGeneModal'
+import { ButtonLink } from 'shared/components/StyledComponents'
+import { getLocus } from 'shared/components/panel/variants/VariantUtils'
+
+const RNA_SEQ_SPLICE_NUM_FIELDS = ['zScore', 'pValue', 'deltaPsi']
+const RNA_SEQ_SPLICE_DETAIL_FIELDS = ['type', 'readCount', 'rareDiseaseSamplesWithJunction', 'rareDiseaseSamplesTotal']
+
+const RNA_SEQ_SPLICE_COLUMNS = [
+  {
+    name: 'junctionLocus',
+    content: 'Junction',
+    width: 4,
+    format: (row, isExport, { onClickCell, familyGuid }) => (
+      <div>
+        <ButtonLink onClick={onClickCell(row, familyGuid)}>
+          {row.junctionLocus}
+        </ButtonLink>
+        {familyGuid && (
+          <GeneSearchLink
+            buttonText=""
+            icon="search"
+            location={`${row.chrom}:${Math.max(1, row.start - RNASEQ_JUNCTION_PADDING)}-${row.end + RNASEQ_JUNCTION_PADDING}`}
+            familyGuid={familyGuid}
+          />
+        )}
```

Comment:
Use the `familyGuid` prop to turn off the search link for the outlier label popup. Maybe, I should add it in another PR and use a different prop name such as `noSearchLink`.

Reply:
You should do the props handling in this PR, it does not need to be separated out into a different PR. Just use a better named prop here in the first place

Reply:
You should do the prop assignment cleanly in this PR, not as a separate PR. I don't understand why this is conditional at all -in what case are we instantiating this without a family?

Reply:
please keep this simple for the purpose of this PR. There is a chance you will need to share some behavior with the upcoming PR and modify behavior accordingly, but I would rather review whats actually needed in this case, and then review the changes we need to support a different case when the time comes, instead of trying to future proof this implementation. I have a feeling this will not end up being the exact place we end up sharing code, so I would rather we not add the configurability here.